bulldozer/merge_test.go

```diff
@@ -15,7 +15,12 @@
 package bulldozer
 
 import (
+	"bytes"
 	"context"
+	"fmt"
+	"github.com/google/go-github/v47/github"
```

Comment:
This is the formatting error - it wants this import to be part of non-stdlib block starting on L26.